packages/typespec-vscode/package.json

```diff
@@ -208,6 +336,9 @@
     "test:e2e": "pnpm test:web",
     "test:web": "vscode-test-web --extensionDevelopmentPath=. --headless --extensionTestsPath=dist/test/web/suite.js ./test/web/data"
   },
+  "dependencies": {
+    "@typespec/compiler": "workspace:~"
```

Comment:
Why would you need this? All needed should have been included by vite

Reply:
removed